tests/integrations/django/test_basic.py

```diff
@@ -518,6 +518,22 @@ def test_does_not_capture_403(sentry_init, client, capture_events, endpoint):
     assert not events
 
 
+def test_render_spans(sentry_init, client, capture_events, render_span_tree):
+    sentry_init(
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=1.0,
+    )
+
+    for url in (reverse("template_test"), reverse("template_test2")):
+        events = capture_events()
+        _content, status, _headers = client.get(url)
+        transaction = events[0]
+        assert(
+            '- op="django.render": description="user_name.html"'
```

Comment:
I would do a strict equality check here. You can acquire the value by first doing `assert x == ''` and copying the error message

Reply:
it was my first implementation but the message is a bit heavy and have not the same indent so it would require two big strings and it would break on new automatic integrations. This implementation is more permissive which can be good or bad. I'll follow your suggestion.

Reply:
I did not implemented this one yet as I gave my point of view, just tell me I did not convince you and I'll comply.

Reply:
https://github.com/getsentry/sentry-python/pull/957/commits/3403adfeffdb49888da131a53fb02dac07e4c471#diff-18f1bd8241119d62eddf36e6bf69d0f29bd8968bf2aca21d9d6238040c60189bR522